packages/styles/combobox.css

```
@@ -91,10 +91,11 @@
 
 .Combobox__input:focus-within {
   border: 1px solid var(--combobox-input-border-focus-color);
-  box-shadow: 0 0 0 2px var(--combobox-input-border-focus-color),
-    0 0 5px var(--combobox-input-border-focus-glow-color),
-    inset 0 1px 2px rgba(0, 0, 0, 0.05);
+  box-shadow: 0 0 0 1px var(--combobox-input-border-focus-color),
+    0px 0px 5px var(--combobox-input-border-focus-glow-color),
```

Nit: 0 values can be unitless in css.

```suggestion
    0 0 5px var(--combobox-input-border-focus-glow-color),
```